homeassistant/components/qbittorrent/helpers.py

```diff
@@ -10,3 +13,48 @@ def setup_client(url: str, username: str, password: str, verify_ssl: bool) -> Cl
     # Get an arbitrary attribute to test if connection succeeds
     client.get_alternative_speed_status()
     return client
+
+
+def seconds_to_hhmmss(seconds) -> str:
+    """Convert seconds to HH:MM:SS format."""
+    if seconds == 8640000:
+        return "None"
+
+    minutes, seconds = divmod(seconds, 60)
+    hours, minutes = divmod(minutes, 60)
+    return f"{int(hours):02}:{int(minutes):02}:{int(seconds):02}"
+
+
+def format_unix_timestamp(timestamp) -> str:
+    """Format a UNIX timestamp to a human-readable date."""
+    dt_object = datetime.fromtimestamp(timestamp, tz=UTC)
+    return dt_object.strftime("%Y-%m-%dT%H:%M:%S%z")
```

Comment:
What's the difference between the manual formatting and the output of the `isoformat` method? Why do we format it at all?